src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -2353,6 +2357,15 @@ public function get_item_schema() {
 				'context'     => array( 'edit' ),
 				'readonly'    => true,
 			);
+
+			$schema['properties']['class_list'] = array(
+				'description' => __( 'An array of the class names for the post container element.' ),
+				'type'        => 'array',
+				'context'     => array( 'view', 'edit' ),
```

Comment:
I'm not sure about the context.
It seems that 'view' and 'edit' would be appropriate for this field, but I could use some help here. I'd be happy to update the PR if these contexts are not correct for the given field.

Reply:
Yeah `view` and `edit` looks correc to me. Basically everything should have `edit`, and `view` should be used if it's a non-privileged field.

The `embed` context is a bit of a bar to meet. It should be used when developers would often need this information when including the resource as an embed. That doesn't seem the case to me here, since the class list would most often be used when looping over the list of posts as a main query.

Reply:
Thank you for the information. It's indeed very helpful in understanding when to use the embed context.